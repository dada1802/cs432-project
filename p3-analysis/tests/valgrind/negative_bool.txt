==698372== Memcheck, a memory error detector
==698372== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==698372== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==698372== Command: ../decaf inputs/negative_bool.decaf
==698372== 
Type mismatch: int expected but bool found on line 4
Type mismatch: bool is incompatible with int on line 4
==698372== 
==698372== HEAP SUMMARY:
==698372==     in use at exit: 0 bytes in 0 blocks
==698372==   total heap usage: 1,877 allocs, 1,877 frees, 228,628 bytes allocated
==698372== 
==698372== All heap blocks were freed -- no leaks are possible
==698372== 
==698372== For lists of detected and suppressed errors, rerun with: -s
==698372== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
