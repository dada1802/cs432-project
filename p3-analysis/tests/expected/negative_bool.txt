Type mismatch: int expected but bool found on line 4
Type mismatch: bool is incompatible with int on line 4
